charts/trento-server/charts/trento-wanda/values.yaml

```diff
@@ -84,6 +84,8 @@ ingress:
           pathType: ImplementationSpecific
         - path: /api/v3/checks
           pathType: ImplementationSpecific
+        - path: /api/v1/groups
```

Comment:
To be compliant with the redirection, maybe we should add the `/api/groups` as well.
This path is redirected to `v1`, right?

Reply:
correct 04734783fa32ecea66d78a4aa5a5e3abb3ba96e8